CesiumGS/cesium
# Add tdd-style npm command

While developing two PR's I found it hard to work in TDD style since there was no ability to run the tests in `watch` mode. This PR adds this ability.
- Added npm script `test:tdd`
- Added gulp `tdd` variable
- Use the `tdd` variable in karma.conf in order to set autoWatch and allow multiple runs
- Make sure no page reloads happen during tests
